EpsilonCCS/Src/CanParsers/MpptCanParser.c

```diff
@@ -0,0 +1,35 @@
+#include "cmsis_os.h"
+
+#include "MpptCanParser.h"
+
+#include "MpptData.h"
+
+extern struct MpptData mpptData[NUM_OF_MPPTS];
```

Comment:
mpptData is already defined within MpptData.h. It is rather strange that it needs to be externed here as well. Try removing it because I think it might compile without it. This also goes for other the externs within other *Parser.c files.